a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -289,7 +289,13 @@ def get_create_rack_vthunder_load_balancer_flow(self, vthunder_conf, topology, l
 
         lb_create_flow.add(lifecycle_tasks.LoadBalancerIDToErrorOnRevertTask(
             requires=constants.LOADBALANCER_ID))
-
+        lb_create_flow.add(database_tasks.ReloadLoadBalancer(
+            name=f_name + '-' + 'reload_loadbalancer',
```

Comment:
We shouldn't need a name on this task especially since we aren't explicitly naming other tasks